packages/lib/shared/hooks/useAprTooltip.ts

```diff
@@ -33,14 +34,20 @@ In addition, veBAL holders can get an extra boost of up to 2.5x.`
 
 const stakingTokenTooltipText = '3rd party incentives (outside the veBAL system)'
 
+const maBeetsVotingRewardsTooltipText =
+  'To receive Voting APR you must vote for incentivized pools in the bi-weekly gauge vote. APR is dependent on your vote distribution.'
+
+const maBeetsRewardTooltipText = 'This is the APR you will receive when a relic is fully matured.'
+
 // Types that must be added to the total base
 const TOTAL_BASE_APR_TYPES = [
   GqlPoolAprItemType.SwapFee_24H,
   GqlPoolAprItemType.IbYield,
   GqlPoolAprItemType.Staking,
   GqlPoolAprItemType.Merkl,
-  GqlPoolAprItemType.Surplus,
+  GqlPoolAprItemType.Surplus_24H,
```

Comment:
Why is this changing? Are you sure it won't break the CoW APRs?

Reply:
the prop is deprecated in the api so i updated it

![image](https://github.com/user-attachments/assets/74755c93-359c-4495-b11d-70f413da73ae)